examples/cluster/plot_kmeans_digits.py

```diff
@@ -53,14 +53,13 @@
 
 
 print(79 * '_')
```

Comment:
also i believe changing the spacing would make this line too short...but I'm not sure if it's worth to change the length. PEP8 doesn't really apply to stdout (although one could argue it fits on 80x25 terminals, but the text already is greater than 79 chars). Are there any other considerations to take into account here? I don't think it would cause any problems on the generated docs as well...